### PR TITLE
Update reader questions with email submission

### PR DESCRIPTION
**Pull request by regiskuckaertz**
Substitute your run-of-the-mill email submission for the notification test.

Pending
- [x] Design from Ben
- [x] Article configuration with URL for XHR
- [x] Final texts

### CODE REVIEW

File: src/main/resources/storyquestions/article/index.fjs [hunk 1]
```diff
@@ -104,16 +63,19 @@ export default ({
...
 
   const onVote = (s: StoryQuestions) => (q: Question): void => {
     record(s.atomId, Actions.VOTE, q[0], q[1]);
   };
 
-  const onSubscribe = (s: StoryQuestions) => (n: Notification): void => {
-    record(s.atomId, Actions.SUBSCRIBE, null, n);
+  const onSubmit = (s: StoryQuestions, q: Question, d: FormData): void => {
+    record(s.atomId, Actions.SUBSCRIBE, q[0], q[1]);
+    const req = new XMLHttpRequest();
```
Comment (regiskuckaertz) [inline src/main/resources/storyquestions/article/index.fjs]: This should be provided by the platform.

File: src/main/scala/AtomRenderer.scala [hunk 2]
```diff
@@ -29,7 +29,7 @@ trait AtomRenderer {
   type JS = Option[String]
 
   def getHTML[A](atom: Atom, data: A, conf: Conf)(implicit reader: Rendering[A]): HTML =
-    reader.html(atom, data).toString
+    reader.html(atom, data)(conf.asInstanceOf[reader.Conf]).toString
```
Comment (regiskuckaertz) [inline src/main/scala/AtomRenderer.scala]: I don't know another way of reconciling the two types. I tried with an implicit evidence that `Conf <:< reader.Conf` but the compiler cannot prove it.

File: src/main/resources/fragments/icons/envelope.scala.html [hunk 3]
```diff
@@ -0,0 +1,4 @@
+@(classNames: List[String] = Nil) 
+@svg("enveloppe", 44, 32, viewBox = true, classNames = classNames){
```
Comment (hosamaly) [inline src/main/resources/fragments/icons/envelope.scala.html]: The file name's spelling is different from the spelling here 😉

Comment (regiskuckaertz) [reply src/main/resources/fragments/icons/envelope.scala.html]: nice catch 👍

File: src/main/scala/AtomRenderer.scala (hunk 2 above)
Comment (tomrf1) [reply src/main/scala/AtomRenderer.scala]: as discussed, this can probably be fixed by refactoring to remove the need for an implicit param

**Comment by tomrf1** [comment]
👍

**Pull request merged**